rpm/felix.spec

```diff
@@ -3,7 +3,7 @@
 Name:           felix
 Summary:        Project Calico virtual networking for cloud data centers
 Version:        2.0.0
-Release:        0.3.beta3%{?dist}
+Release:        0.1.rc2%{?dist}
```

Comment:
This has gone backwards from `0.3.beta3` so I think the new package will be a downgrade.